source/develop/release-management/features/external-network-provider.html.md

```diff
@@ -0,0 +1,376 @@
+---
+title: External Network Providers
+category: feature
+authors: mmirecki
+wiki_category: Feature
+wiki_title: Features/External Network Providers
+wiki_revision_count: 1
+wiki_last_updated: 2016-01-25
+feature_name: External Network Providers
+feature_modules: engine,vdsm
+feature_status: In Development
+---
+
+# External Network Providers
+
+
+## Summary
+
+The purpose of this feature is to enable support for networks not defined by oVirt, but supplied by an external provider.
+
+
+### Owner
+
+*   Feature Owner: Marcin Mirecki: [ mmirecki](User:mmirecki)
+
+<!-- -->
+
+*   Email: <mmirecki@redhat.com>
+
+### Benefit to oVirt
+
+oVirt will be able to use networks definded by external providers.
+oVirt will provide an API for the external providers allowing to easily integrate with oVirt.
+
+
+## Detailed Description
+
+Many organizations use centralized network management systems to handle all their networking, and would like them to cover their oVirt managed environments. Currently the only option of using external networks in oVirt is to used the OpenStack Neutron (OSN) integration, which allows the import of OSN networks and the provisioning of VM's connected to these networks. This feature is however very specific to OpenStack Neutron. We would like to open up this API to other external network providers, by making it simpler, more general and less dependant on OSN features. The result should be an API over which oVirt can communicate with external network management systems, and use the networks defined in them in provisioned VMs.
+
+For information about OSN integration please refer to:
+[OSN Integration](http://www.ovirt.org/develop/release-management/features/network/osn-integration/)
+[Detailed OSN Integration](http://www.ovirt.org/develop/release-management/features/network/detailed-osn-integration/)
+
+### Differences to OVN Integration
+
+*   removal of UI features related to OVN, such as tenants
```

Comment:
We will not be removing these things. We need to be able to hide them and expose them only when using a Neutron provider.